src/AvroConvert/AvroConvert.DeserializeContainer.cs

```diff
@@ -0,0 +1,146 @@
+﻿#region license
```

Comment:
The default AvroConvert.Deserialize() method handles objects serialized in this way correctly. Could you please revert the changes done in the deserialization part?